modules/project/main.tf

```diff
@@ -47,9 +47,17 @@ locals {
   parent_id   = var.parent == null ? null : split("/", var.parent)[1]
   prefix      = var.prefix == null ? "" : "${var.prefix}-"
   project = (
-    var.project_create
-    ? try(google_project.project.0, null)
-    : try(data.google_project.project.0, null)
+    var.project_create ?
```

Comment:
Let me know if you prefer the conditional in each field instead of a single one for the whole object.

Reply:
single conditional is easier to read IMO